docfx_project/docfx.json

```diff
@@ -87,7 +87,7 @@
         "branch": "master",
         "apiSpecFolder": "apidoc"
       }
-      , "_github_link":"https://github.com/openziti"
+      , "_github_link":"https://github.com/openziti/ziti-doc"

```

Comment:
Is this the link in the top right?  I kinda like it better going to org root than down into the ziti-doc repo.  The 'Improve this doc' looks right...

Reply:
It's an option, sure... Before you would get linked directly to the page you were viewing. I'll leave this open for a bit to see if there's any other opinions in either direction.

Reply:
'Improve this doc' already links directly to the page you are viewing.